common/djangoapps/student/tests/tests.py

```diff
@@ -18,6 +18,7 @@
 from django.core.urlresolvers import reverse
 from django.test import TestCase
 from django.test.client import Client
+from django.test.utils import override_settings
```

Comment:
Nit: since Django 1.7 `override_settings` should be imported from `django.test`.